### PR TITLE
add gameplay readme

### PR DESCRIPTION
**Pull request by kfu02**
## Description
Adds basic gameplay README to explain the most-used files and folders in rj_gameplay/.

### CODE REVIEW

File: rj_gameplay/README.md [hunk 1]
```diff
@@ -1,2 +1,73 @@
...
+
+Atomic robot behavior which consists of a behavior tree and calls a ROS action.
+(Examples: Move, Kick)
+
+**Tactic - `rj_gameplay/tactic/**
+
+Handles complex single robot behavior. Think of as one role in a play.
+(Examples: Goalie Receiver Waller)
+
+**Play - `rj_gameplay/play/**
```
Comment (oswinso) [inline rj_gameplay/README.md]: Missing closing backtick here.

image

**Review by oswinso — APPROVED** [review]

**Review by kasohrab — APPROVED** [review]
other than oswin's comment LGTM!

File: rj_gameplay/README.md (hunk 1 above)
Comment (kfu02) [reply rj_gameplay/README.md]: fixed, ty

**Comment by kfu02** [comment]
restoring to show CI issues

**Comment by kfu02** [comment]
nvm

**Pull request merged**